AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst

```diff
@@ -0,0 +1,114 @@
+.. _Weather constraints: https://obs-ops.lsst.io/Observing-Constraints/AuxTel-Observing-Constraints/AuxTel-Weather-Constraints-index.html
```

Comment:
Not sure you want to leave this link not linked to the file tree.

Reply:
I have added an internal reference in the  bullet point 
- :ref:`Weather constraints <Observing-Constraints-AuxTel-Weather-Constraints>`